lib/irc.js

```diff
@@ -654,6 +655,9 @@ Client.prototype.connect = function(retryCount, callback) { // {{{
                     self.conn.authorizationError === 'CERT_HAS_EXPIRED') {
                     util.log('Connecting to server with expired certificate');
                 }
+                if (typeof self.opt.webirc.ip !== 'undefined') {
```

Comment:
This might be better as `if (self.opt.webirc.ip) { `. Currently this would run if an IP was an empty string as well.